README.md

```diff
@@ -5,21 +5,37 @@
 
 Run Selenium tests more easily with automatic installation and updates for all supported webdrivers.
 
-## Update: Selenium Manager
+## Update: Future of this Project
 
-Selenium is working to make all 3rd party driver managers obsolete with a single solution that works in all
-languages. The new Selenium Manager is [still in beta](https://github.com/orgs/SeleniumHQ/projects/5), 
-but if you are using Selenium 4.6+ you may not need this gem.
+With Google's new [Chrome for Testing](https://developer.chrome.com/blog/chrome-for-testing/) project, 
+and Selenium's new [Selenium Manager](https://www.selenium.dev/documentation/selenium_manager/) feature, 
+what is required of this gem has changed..
 
-The current logic (as of Selenium 4.8) determines driver path in this order:
-* Specified in a `Service` constructor provided as an argument in the `Driver` constructor
-* Specified by `driver_path` attribute accessor for the `Service` class (which is what this gem does)
-* Located in a directory included in the `PATH` environment variable
-* Located in the directory Selenium Manager has downloaded it to
+If you can update to to the latest version of Selenium, please do so and stop requiring this gem.
+Selenium 4.10 will work for Chrome v115 (ignore the warning), and Selenium 4.11 will work for everything going forward.
+Please provide feedback or raise issues with [Selenium Project](https://github.com/SeleniumHQ/selenium/issues/new/choose)
 
-If you don't have any drivers in locations on `PATH`, you can try not requiring this gem and see if you still need it.
+If you cannot upgrade to the latest version of Selenium, you can set the required version of chromedriver to v114
+(`Webdrivers.required_version = '114.0.5735.90'`) and 
```

Comment:
@titusfortner I think this line of code is wrong - or it was for me.

I used the line `Webdrivers::Chromedriver.required_version = '114.0.5735.90'`

Reply:
oops, you're right.

Reply:
fixed, thanks.